### PR TITLE
Clarifying which username/password. 

### PR DESCRIPTION
**Pull request by mary-thoughtspot**
@tushar-thoughtspot updates related to the username/password
Tweaks to usability of ODBC instructions

Signed-off-by: mary.anthony <mary.anthony@thoughtspot.com>

### CODE REVIEW

File: _data-integrate/reference/simba-settings.md [hunk 1]
```diff
@@ -175,15 +175,15 @@ The following tables summarize the configuration properties.
          </td>
       </tr>
       <tr>
-         <td><code>UID</code></td>
-         <td>Login Username</td>
-         <td>The ODBC/JDBC login username.
+         <td><code>username</code></td>
+         <td>UID</td>
+         <td>Part of a user username/password combination. This combination should correspond to a ThoughtSpot application user with permissions appropriate to your ETL requirements.  Typically, this user is an administrator user on the application.
```
Comment (tushar-thoughtspot) [inline _data-integrate/reference/simba-settings.md]: This user is user with data management or administrator privileges.

File: _data-integrate/reference/simba-settings.md [hunk 2]
```diff
@@ -175,15 +175,15 @@ The following tables summarize the configuration properties.
...
          </td>
       </tr>
       <tr>
-         <td><code>PWD</code></td>
-         <td>Login Pasword</td>
-         <td>The ODBC/JDBC login password.
+         <td><code>password</code></td>
+         <td>Password</td>
+         <td>Part of a user username/password combination. This combination should correspond to a ThoughtSpot application user with permissions appropriate to your ETL requirements.  Typically, this user is an administrator user on the application.
```
Comment (tushar-thoughtspot) [inline _data-integrate/reference/simba-settings.md]: This user is user with data management or administrator privileges.

File: _data-integrate/clients/use-jdbc-driver.md [hunk 3]
```diff
@@ -14,8 +14,8 @@ To use the JDBC driver, include the JDBC library in your path, and provide the c
 |Server IP address | The ThoughtSpot appliance URL or IP address. The IP address can be found by going to `http://<server-ip\>:2201/status/service?name=simba_server`|
 |Simba port | The simba port, which is 12345 by default.|
 |Database name | This is not the machine login username. The ThoughtSpot Database name to connect to.|
-|Database username | The name of a ThoughtSpot user with administrator permissions.|
-|Database password | This is not the machine login password. The ThoughtSpot user password.|
+|username | The name of a ThoughtSpot user with administrator permissions.|
```
Comment (tushar-thoughtspot) [inline _data-integrate/clients/use-jdbc-driver.md]: With data management or administrator privilege.

File: _data-integrate/clients/install-odbc-windows.md [hunk 4]
```diff
@@ -45,9 +50,27 @@ To obtain and install the ODBC driver for Windows:
...
+You can also [add a new ODBC data source]({{ site.baseurl
+}}/data-integrate/clients/add-new-ODBC-source-windows.html). This capability
+supports connecting to multiple ThoughtSpot instances.
+
+
+## Testing your ODBC connection
+
+At this point, you can test your ODBC connection to ThoughSpot.  It is important
+to recall that the username/password you use belongs to a ThoughSpot application
+user with with administrator permissions.  Before trying the ODBC connection,
```
Comment (tushar-thoughtspot) [inline _data-integrate/clients/install-odbc-windows.md]: data management or administrator permissions.

**Comment by mary-thoughtspot** [comment]
Thanks @tushar-thoughtspot

**Pull request merged**